blog/2024-10-8-self-hosting-reflex-with-docker.md

```diff
@@ -205,6 +205,7 @@ server {
  server_name frontend;
 
 
+ error_page   404  /404.html;
```

Comment:
```suggestion
 error_page   404  /usr/share/nginx/html/404.html;
```

Reply:
we want the app's `/404.html` page because that's where the dynamic route client side redirect logic lives